.scribe/intro.md

```diff
@@ -3,7 +3,7 @@
 The documentation
 
 <aside>
-    <strong>Base URL</strong>: <code>https://staging-api.learnhub.mk</code>
```

Comment:
Please use the APP_STAGING_URL environment variable so that it generates the correct url every time and the API calls in the docs work as expected.